appcheck/firebase-appcheck/src/main/java/com/google/firebase/appcheck/internal/DefaultAppCheckToken.java

```diff
@@ -138,10 +138,10 @@ public static DefaultAppCheckToken constructFromRawToken(@NonNull String token)
     Map<String, Object> claimsMap = TokenParser.parseTokenClaims(token);
     long iat = getLongFromClaimsSafely(claimsMap, ISSUED_AT_KEY);
     long exp = getLongFromClaimsSafely(claimsMap, EXPIRATION_TIME_KEY);
-    long expiresInMillis = exp - iat;
+    long expiresInMillis = (exp - iat) * 1000L;
```

Comment:
Nit: if might want 1000L as a constant, as are in tests. But, like, eh - not a big deal as is.

Reply:
Done.